### PR TITLE
Added pagination component and docs

### PR DESCRIPTION
**Pull request by matine**

### CODE REVIEW

File: src/templates/pages/pagination.hbs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            </div>
+        </section>
+
+
+        <!-- PAGINATION MARKUP -->
+        <section id="pagination-markup">
+            <header class="content-block bg-light-grey">
+                <div class="container">
+                    <h3>The markup</h3>
+                    <p>Based on the standard Laravel pagination markup.</p>
```
Comment (antpuleo2586) [inline src/templates/pages/pagination.hbs]: I think it's actually the Bootstrap markup, that's where Laravel gets it from: https://laravel.com/docs/5.5/pagination#customizing-the-pagination-view

File: src/templates/layouts/default.hbs [hunk 2]
```diff
@@ -23,6 +23,8 @@
 		</div>
 	<![endif]-->
 
+	<a href="https://github.com/Pod-Point/pod-point-ui-toolkit"><img style="position: absolute; top: 0; right: 0; border: 0;" src="https://camo.githubusercontent.com/365986a132ccd6a44c23a9169022c0b5c890c387/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f7265645f6161303030302e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_red_aa0000.png"></a>
```
Comment (antpuleo2586) [inline src/templates/layouts/default.hbs]: Maybe using nesting so this line isn't as long?

Comment (matine) [reply src/templates/layouts/default.hbs]: It's just a copy and paste jobby from the GitHub ribbons website https://github.com/blog/273-github-ribbons

File: src/templates/pages/pagination.hbs (hunk 1 above)
Comment (matine) [reply src/templates/pages/pagination.hbs]: Ah, thanks!

**Pull request merged**